static/scripts/loggedin.js

```diff
@@ -59,16 +70,57 @@ function adjustContentWidth(sidebar) {
     }
 }
 
+function toggleSidebarOnWindowWidth() {
+    if (overlay) {
+        if (window.innerWidth <= 1279) {
+            sidebar.classList.remove('visible');
+            sidebar.classList.add('hidden');
+        }
+        if (window.innerWidth >= 1280 && sideBarExtendedValue == 'true') {
+            sidebar.classList.remove('hidden');
+            sidebar.classList.add('visible');
+            toggleInTopbar.classList.add('invisible-toggle');
+            toggleInSidebar.classList.remove('invisible-toggle');
+        }
+        if (window.innerWidth >= 1280 && sideBarExtendedValue == 'false') {
+            sidebar.classList.remove('visible');
+            sidebar.classList.add('hidden');
+            toggleInSidebar.classList.add('invisible-toggle');
+            toggleInTopbar.classList.remove('invisible-toggle');
+        }
+        if (sidebar.classList.contains('hidden')) {
+            overlay.style.display = 'none';
+        }
+
+        adjustContentWidth();
+    }
+}
+
+function adjustContentWidthOnLoad() {
+    if (contentWrapper) {
+        if (sideBarExtendedValue == 'false') {
+            contentWrapper.style.paddingLeft = '0px';
+            sidebar.classList.add('hidden');
+        } else {
+            contentWrapper.style.paddingLeft = '255px';
+            sidebar.classList.remove('hidden');
+        }
+    }
+    toggleSidebarOnWindowWidth();
+}
+
+
+
 function toggleSidebar() {
-    const sidebar = document.querySelector('.sidebar');
-    const overlay = document.querySelector('.overlay');
-    const contentDiv = document.querySelector('.content-min-height');
     const sidebarItems = document.querySelectorAll('.sidebar-item');
     const sidebarSubitems = document.querySelectorAll('.subitem');
     const legalLinks = document.querySelectorAll('.legal-link');
 
-	if (sidebar) {
-		if (sidebar.classList.contains('hidden')) {
+    window.localStorage.setItem('sidebarExpanded', sidebar.classList.contains('hidden'));
+    window.location.reload();
```

Comment:
We shouldn't reload the page when toggling the sidebar.
I also tested it and it causes problems on mobile where the sidebar closes automatically when you open it.

We need to update `sidebarExtendedValue` whenever we save the value to the local storage. 

Maybe something like the following would work 🤔  :

```js
let sidebarExtendedValue = window.localStorage.getItem('sidebarExpanded');

function setSidebarStateExtendenValue(newValue) {
    const newValueString = JSON.stringify(newValue);
    window.localStorage.setItem('sidebarExpanded', newValueString);
    sidebarExtendedValue = JSON.stringify(newValueString);
}

if (!sidebarExtendedValue) {
    setSidebarStateExtendenValue(true);
}
```